### PR TITLE
Update cd.yml

### PR DESCRIPTION
**Pull request by qdrn**
This should work. Removes the need from the custom image.

### CODE REVIEW

**Comment by yvan-sraka** [comment]
For more consistence, we should rather choose between, for all artifacts:

- build using [cross-compilation](https://github.com/massalabs/massa/blob/fix-cd/.github/workflows/cd.yml#L45-L62) (what are done in `windows` job)
- build using [host image target](https://github.com/massalabs/massa/blob/fix-cd/.github/workflows/cd.yml#L76-L77) (what are done in `linux` and `macos` jobs)

I advise the second solution since it will be faster (if I understand it well) because it will use the cargo build cache of test jobs (thanks to https://github.com/marketplace/actions/rust-cache)

**Comment by qdrn** [comment]
Yes I agree with you. I made a quick fix because it's easy to make mistakes when manually compiling and uploading.

I agree that second solution sounds better.

Regarding caching, I'm not sure it works well as it always took quite a while to compile. I may be wrong.

**Comment by qdrn** [comment]
I suggest that if this works we merge this issue and open a follow up to make a better and more consistent CD. Maybe use job matrix syntax ? What do you think ?

**Review by yvan-sraka — APPROVED** [review]
That's neat ✨

**Comment by qdrn** [comment]
Thanks @yvan-sraka !

I pretty sure that cache is never used in the CD/CI. I can't figure why. For example if you look at https://github.com/massalabs/massa/runs/4781494606?check_suite_focus=true which happened just after a commit that only changed the `cd.yml` file, the cache is not found. Any idea ?

**Comment by AureliaDolo** [comment]
bors merge

**Review by adrien-zinger — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Comment by yvan-sraka** [comment]
bors merge

**Review by yvan-sraka — APPROVED** [review]

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -12,8 +12,21 @@ env:
 jobs:
   # The 3 following build jobs are 90% copy-pasted and should be refactored using
```
[the remark this answers is not in the document]
Comment (qdrn) [reply .github/workflows/cd.yml:13]: Will do.

**Comment by qdrn** [comment]
I think at some point you wanted to add the `RELEASE_VERSION` in the artifacts names @yvan-sraka. I'm not familiar with Github actions and struggled a bit to do that so if you can do it feel free to do it.

**Pull request merged**